rpc/handler.go

```diff
@@ -569,51 +555,6 @@ func (h *handler) runMethod(ctx context.Context, msg *jsonrpcMessage, callb *cal
 	return nil
 }
 
-var nullAsBytes = []byte{110, 117, 108, 108}
```

Comment:
no need to write nil if callback failed? just to clarify

Reply:
nope, `writeNilIfNotPresent` operates only on streams and the stack-based json stream takes exactly care of writing nil or closing pending objects, if necessary